motor_control_node:
  ros__parameters:
    cmd_vel_topic: "/cmd_vel"
    wheel_base: 0.5
    wheel_diameter: 0.1
    max_rpm: 100
    min_rpm: 10
    motor_serial_device: "/dev/serial0"

